packages/evm/src/fee/TwapFee.ts

```diff
@@ -0,0 +1,40 @@
+import { FeeHandlerType } from '@buildwithsygma/core';
+import { TwapGenericFeeHandler__factory } from '@buildwithsygma/sygma-contracts';
+import { utils } from 'ethers';
+
+import type { EvmFee } from '../types.js';
+
+import { BaseEvmTransferFeeCalculator } from './types.js';
+import type { EvmFeeCalculationParams } from './types.js';
+
+export class TwapFeeCalculator extends BaseEvmTransferFeeCalculator {
+  constructor() {
+    super();
+  }
+
+  async calculateFee(_params: EvmFeeCalculationParams): Promise<EvmFee> {
```

Comment:
do we need to use this `_varName`naming convention here and below? 🤔

Reply:
so the reason I am using `_varName` convention here is because we are using protected variables and getter functions, so if I have a property `name` a getter function for name would be `get name()` and therefore I can't use `name` for the property itself as it has been used by the getter function

Reply:
I have the impression this was the reason